src/js/components/tablero/NumberCard.jsx

```diff
@@ -1,26 +1,34 @@
 import React from 'react';
+import { SortableElement, sortableHandle } from 'react-sortable-hoc';
 import PropTypes from 'prop-types';
 import './tablero.scss';
 
 function truncateString(value, length) {
   return (value.length > length) ? `${value.substr(0, length - 1)}...` : value;
 }
 
-const NumberCard = ({
+const DragHandle = sortableHandle(() => (
+  <span className="dragHandler">::</span>
```

Comment:
Why not using a font icon for this, here is nice fontawesome icon that could be used here: https://fontawesome.com/v4.7.0/icon/ellipsis-v (here is example of it)
![Selection_182](https://user-images.githubusercontent.com/12113312/80089709-a03d1880-855e-11ea-8920-02b3054bff5b.png)